apps/desktop/package.json

```diff
@@ -81,10 +82,10 @@
 		"tinykeys": "^2.1.0",
 		"ts-node": "^10.9.2",
 		"vite": "catalog:",
-		"vitest": "^2.0.5",
-		"@types/node": "^22.3.0"
+		"vitest": "^2.0.5"
 	},
 	"dependencies": {
+		"@anthropic-ai/sdk": "^0.27.3",
 		"openai": "^4.47.3"
```

Comment:
I have to admit, it's a bit unclear what the difference is between our use of `dependencies` vs. `devDependencies`. For consistency, perhaps we should move both of these? @ndom91 in case you know why we're doing it this way.

Reply:
I think the standard way of ordering the NPM dependencies is to add anything that gets bundled into the distributable under `dependencies` and everything not distributed (all the local dev dependencies, test dependencies, linters, etc.) under `devDependencies`.
So I think this change should be conformant.

https://docs.npmjs.com/specifying-dependencies-and-devdependencies-in-a-package-json-file#skip-to-content